src/components/views/rooms/RoomSublist.tsx

```diff
@@ -752,6 +753,7 @@ export default class RoomSublist extends React.Component<IProps, IState> {
                                 >
                                     <span className={collapseClasses} />
                                     <span>{this.props.label}</span>
+                                    { visibleRooms.length > 0 &&  `(${visibleRooms.length})`}
```

Comment:
To avoid changing the default element behaviour, please add a config value, which defaults to false, that controls if this is shown

Reply:
done